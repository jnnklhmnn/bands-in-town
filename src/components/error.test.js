import React from 'react';
import Error from './Error.js';
import renderer from 'react-test-renderer';

test('Error should match Snapshot', () => {
    const component = renderer.create(
        <Error/>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});