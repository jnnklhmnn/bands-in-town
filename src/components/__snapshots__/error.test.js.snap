// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match Snapshot 1`] = `
<div
  className="contentbox"
>
  <div>
    <h3>
      Something went terribly wrong, please try again.
    </h3>
  </div>
</div>
`;
